lib/rdstation/error/formatter.rb

```diff
@@ -102,6 +103,14 @@ def from_hash_of_arrays
           array_of_errors.push(*errors)
         end
       end
+
+      def error_details
+        {
+          max: @headers.fetch('ratelimit-limit-quotas', 0),
+          usage: @headers.fetch('ratelimit-limit-quotas', 0),
```

Comment:
Have the `usage` and `max`  keys the same value?

Reply:
I know it's weird, but in this case yes they do 😁
I talked to @wendeldavid about this and analyzing the rate-limit headers returned we noticed that there are no cases in which the usage is greater than max, because when 429 occurs, these values are equal, and even if you keep calling the API it won't change.